CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributions are welcome!
+
+We do all of NeMo-text-processing's development in the open. Contributions from the open-source community are welcome.
+
+
+# Pull Requests (PR) Guidelines
+
+**Send your PRs to the `main` branch**
+
+1) Make sure your PR does one thing. Have a clear answer to "What does this PR do?".
+2) Make sure you sign your commits. E.g. use ``git commit -s`` when you commit.
+3) Make sure to add test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
+4) Make sure all unittests finish successfully before sending PR:
+   1) ``pytest`` or (if yor dev box does not have GPU) ``pytest --cpu`` from the root folder
+   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
+5) If you are adding a **new** Python file with a license header, the first line needs to be `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` (change `2023` to the current year).
+6) If your `text_normalization/LANG/graph_utils.py` is mainly copied from [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
+7) Add ``__init__.py`` for every folder and subfolder.
+8) Remove import guards (`try import: ... except: ...`) if not already done.
+9) follow codeQL results and remove unused variables and imports (report is at the bottom of the PR in github review box)
+10) Add your language support to [tools/text_processing_deployment/export_grammars.sh](tools/text_processing_deployment/export_grammars.sh) and [tools/text_processing_deployment/pynini_export.sh](tools/text_processing_deployment/pynini_export.sh).
```

Comment:
There's nothing to do for export_grammars.sh; pynini_export.sh should be pynini_export.py

Reply:
thanks, let me change it